src/libpmemfile-posix/dir.c

```diff
@@ -229,10 +229,9 @@ vinode_add_dirent(PMEMfilepool *pfp,
 	struct pmemfile_inode *parent = parent_vinode->inode;
 
 	/* don't create files in deleted directories */
-	if (parent->nlink == 0)
```

Comment:
More specifically: without the extra brackets, it is easy to think this was a mistake during a merge, or during copy-pasting code. Otherwise, I believe it is obvious, that the comments are main reason for having two separate ifs, and I see it as something easier to understand.
But it is not a matter of life-and-death anyways.